lenskit-eval/src/main/java/org/lenskit/eval/traintest/WebAppListener.java

```diff
@@ -0,0 +1,206 @@
+package org.lenskit.eval.traintest;
+
+import com.google.common.base.Stopwatch;
+import com.google.common.eventbus.Subscribe;
+import org.lenskit.util.monitor.JobEvent;
+import org.lenskit.util.monitor.TrackedJob;
+import org.slf4j.Logger;
+import java.net.Socket;
+import java.util.Scanner;
+import java.io.IOException;
+import java.util.Date;
+import java.io.BufferedReader;
+import java.io.DataOutputStream;
+import java.io.InputStreamReader;
+import java.sql.Timestamp;
+import java.net.*;
+
+import java.io.*;
+
+public class WebAppListener {
+    public static Logger log;
+    private static Socket socket;
+    private static PrintStream ps;
+    private static final String USER_AGENT = "Mozilla/5.0";
```

Comment:
Alright, so I should just replace "Mozilla/5.0" with "Lenskit Experiment Runner" ?